bears/js/JSStandardBear.py

```diff
@@ -0,0 +1,25 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='standard',
+        output_format='regex',
+        output_regex=r'  (?P<filename>[^:]+):(?P<line>\d+):(?P<column>\d+):'
+                     r' (?P<message>.+)')
+class JSStandardBear:
+    """
+    JavaScript Standard Style
```

Comment:
Can we have a basic description of what the bear is capable of doing ? Our users might wanna know what this bear has to offer.